api.yml

```diff
@@ -17,13 +17,38 @@ tags:
   - name: public
     description: Public APIs facing the Holder and Verifier.
 paths:
-  /credential:
+  /credentials/issueCredential:
     post:
       tags:
         - internal
       summary: Issues a credential and returns it in the response body.
       operationId: issueCredential
       description: Issues a credential and returns it in the response body. Support of this part of the API is REQUIRED for implementations.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueCredentialRequest'
+        description: Parameters for issuing the credential.
+      responses:
+        '201':
+          description: Credential successfully issued!
+          content:
+            application/json:
+              schema:
+                type: object
+                description: The issued credential. This SHOULD include a reference or ID that can be used later for other parts of the API, e.g. changing and retrieving the status of the credential, or refreshing the credential.
+        '400':
+          description: invalid input!
+        '500':
+          description: error!
+  /credentials/composeAndIssueCredential:
```

Comment:
I agree, lets discuss here: https://github.com/w3c-ccg/vc-issuer-http-api/issues/41

Reply:
Can I resolve this?

Reply:
No discussion happened, can we make this:

```suggestion
  /credentials/compose
```

And presume that `compose` cannot possibly conflict with a reference ID? If we make this endpoint just compose a VC that can then be submitted via `/credentials` for issuance, then this would be a "controller" per the style guide as we're not creating a resource that is stored/referenced on the server. We also get composition and simpler APIs.

Reply:
@peacekeeper -- If we have the `compose` API just compose and you turn around and submit the composed VC to `/credentials` would that work for you? If so, I could quickly agree to this and it would make implementation easier for a number of us, I would think. If not, could we at least call one endpoint: `/credentials` and then we can leave this one as is with the longer name.

Reply:
We can handle these comments in another PR.